Tooll/Components/TextButtonEdit.xaml.cs

```diff
@@ -125,6 +132,14 @@ protected void TextEdit_LostFocus(object sender, RoutedEventArgs e)
             }
             TextEdit.Visibility = Visibility.Collapsed;
         }
+        
+        protected void TextEdit_PreviewMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            if (e.ClickCount == 3)
+            {
+                XTextEdit.SelectAll();

```

Comment:
shouldn't the event be marked as handled and the method be exited with return? Something like...

    e.handled = true;
    return;